novawallet/Modules/Settings/SettingsViewFactory.swift

```diff
@@ -21,13 +21,19 @@ struct SettingsViewFactory {
             quantityFormatter: NumberFormatter.quantity.localizableResource()
         )
 
+        let proxyNotificationService = WalletNotificationService(
```

Comment:
I wouldn't create it everytime. Let's just add one to service coordinator and pass around